MixItUp.Base/Util/SpecialIdentifierStringBuilder.cs

```diff
@@ -366,14 +411,6 @@ public SpecialIdentifierStringBuilder(string text, Guid randomUserSpecialIdentif
                     }
                 }
 
-                IEnumerable<UserViewModel> users = await ChannelSession.ActiveUsers.GetAllUsers();
-                if (users.Count() > 0)
-                {
-                    Random random = new Random();
-                    int index = random.Next(users.Count());
-                    
-                }
-
```

Comment:
I don't know what this code did, but it looks like it does nothing, so... I removed it.